x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/progress_stats.tsx

```diff
@@ -22,6 +22,7 @@ import { DataFrameAnalyticsId } from '../../../../common/analytics';
 
 export const PROGRESS_REFRESH_INTERVAL_MS = 1000;
 const FAILED = 'failed';
+const STOPPED = 'stopped';
```

Comment:
Think both of these might already be defined by DATA_FRAME_TASK_STATE

Reply:
Or alternatively can you use the `JOB_STATE` values from `common/constants/states.ts` ?

Reply:
Good catch - updated in d77fbbbe42d984d53aafe1462a9b5669fbc8a435